src/components/BooksList.jsx

```diff
@@ -1,12 +1,19 @@
 import BookItem from "./BookItem";
-import { useSelector } from 'react-redux';
-
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect,  } from "react";
+import { getBooks } from "../redux/books/booksSlice";
 const BooksList = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getBooks());
+  });

```

Comment:
- [ ] Good job so far 👍🏽 . I can see that your getBooks action is making unlimited network requests. I think you should correctly update the dependency in your useEffect to fix this. e.g

```
  useEffect(() => {
    dispatch(getBooks());
  },[]);
```

Reply:
I have a linter error with the dependencies array empty. should I add dispatch as dependency? or disable the eslint for that line?